test/functional/apps/discover/group3/_request_counts.ts

```diff
@@ -289,7 +291,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await expectSearches(type, 1, async () => {
           await discover.toggleChartVisibility();
         });
-        await expectSearches(type, 3, async () => {
+        await expectSearches(type, 2, async () => {
```

Comment:
ha! 🎼  another requests bites the dust !
and in case this turns green: https://github.com/elastic/kibana/pull/206389
3 more requests will bite the dust

Reply:
I just realized #206389 is a PR and not an issue 🤦 In that case, awesome! These enhancements combined should resolve all chart visibility related fetching issues 🤘